src/apps/create-patron-user-info/CreatePatron.entry.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from "react";
+import { withConfig } from "../../core/utils/config";
+import { withText } from "../../core/utils/text";
+import { withUrls } from "../../core/utils/url";
+import CreatePatron from "./CreatePatron";
+
+interface CreatePatronConfigProps {
+  pincodeLengthMinConfig: string;
+  pincodeLengthMaxConfig: string;
+  blacklistedPickupBranchesConfig: string;
+  branchesConfig: string;
+}
+interface CreatePatronUrlProps {
+  redirectOnUserCreatedUrl: string;
+  fbsBaseUrl: string;
+  publizonBaseUrl: string;
+}
+
+interface CreatePatronTextProps {
+  pickupBranchesDropdownLabelText: string;
+  patronPageChangePincodeHeaderText: string;
+  pickupBranchesDropdownNothingSelectedText: string;
+  patronPageChangePincodeBodyText: string;
+  patronPagePincodeLabelText: string;
+  patronPageConfirmPincodeLabelText: string;
+  patronPagePincodeTooShortValidationText: string;
+  patronPagePincodesNotTheSameText: string;
+  patronContactPhoneLabelText: string;
+  patronContactNameLabelText: string;
+  patronContactInfoBodyText: string;
+  patronContactInfoHeaderText: string;
+  patronContactPhoneCheckboxText: string;
+  createPatronConfirmButtonText: string;
+  createPatronCancelButtonText: string;
+  patronContactEmailLabelText: string;
+  patronContactEmailCheckboxText: string;
+  createPatronChangePickupHeaderText: string;
+  createPatronChangePickupBodyText: string;
+  createPatronHeaderText: string;
+  createPatronInvalidSSNHeaderText: string;
+  createPatronInvalidSSNBodyText: string;
+}
```

Comment:
Just an advise: It is easier for the person that implements the data props in dpl-cms if the props are in alphabetic order.